code/lists/jobs.dm

```diff
@@ -1,3 +1,15 @@
+//Mail Groups by Department
+#define MGD_PARTY "Party Line" //Is the clown the head of the Party department?
+#define MGD_COMMAND "command"
+#define MGD_SECURITY "security"
+#define MGD_MEDBAY "medbay"
+#define MGD_MEDRESEACH "medresearch"
+#define MGD_SCIENCE "science"
+#define MGD_CARGO "cargo"
+#define MGD_BOTANY "botany"
+#define MGD_KITCHEN "kitchen"
+
+
```

Comment:
These defines should prolly be in one of the _ prepended files like _build.dm or something, I think? byond compiles everything in file order, so that means things "before" this .dm can't reference these defines

Reply:
Excellent, I'd never taken the time to look into the _std folder, but I like how's it's all organized, makes a lot of sense. And yes, I understand the "load-order" that you're getting at.